src/Eager.ts

```diff
@@ -4,315 +4,409 @@ export { CoreHelpers } from './ui/Templates/CoreHelpers';
 export { SearchInterface, StandaloneSearchInterface } from './ui/SearchInterface/SearchInterface';
 export { jQueryInstance as $ } from './ui/Base/CoveoJQuery';
 export { underscoreInstance as _ } from './ui/Base/CoveoUnderscore';
+export { AnalyticsUtils } from './utils/AnalyticsUtils';
```

Comment:
The order of export in this file has an impact on the final output.

If you look at the build output in travis, you will see failed UT. 

My guess is that this PR creates some kind of circular dependency issue (either AnalyticsUtils or analyticsActionCauseList)

Reply:
Yeah I just saw that and i'm trying to figure out why..
AnalyticsUtils does import AnalyticsActionListMeta also but normally this shouldn't be a problem 🤔

Reply:
Ok I just found it, the error is caused by the export of Analytics. If I move it at the bottom of Eager.ts it works. But, should I just import what I personnally need instead, which is analyticsActionCauseList, Checkbox and the other form widgets, instead of everything? That might be safer, what do you think @olamothe